### ISSUE TITLE
optional property must have @default documentation

### ISSUE THREAD

**Issue by ren-yamanashi** [ren-yamanashi/eslint-cdk-plugin#99]
### Feature Description

Add a `@default` documentation must be attached to optional properties to be exposed in interface.

### Motivation

By putting `@default` documentation for optional properties, one can understand what the value will actually be if the property is not specified, without having to look at the actual code details

### Invalid Code Examples

```typescript
interface MyConstructProps {
 // ❌ must write `@default`
 /**
  * S3 bucket to be specified for the resourc
  */
  readonly bucket?: IBucket
}
```

### Valid Code Examples

```typescript
interface MyConstructProps {
 /**
  * S3 bucket to be specified for the resource
  * @default no bucket
  */
  readonly bucket?: IBucket
}
```

### Rule Type

problem (possible errors)

### Implementation Details

- [ ] This rule is related to AWS CDK best practices
- [x] I'm willing to submit a PR for this feature

### Additional Context

Similar rules exist for awslint

https://github.com/aws/aws-cdk/blob/main/packages/awslint/lib/rules/docs.ts#L49-L63
